examples/hn.svelte.dev/.gitignore

```diff
@@ -2,4 +2,5 @@
 node_modules
 /.svelte
 /build
-/functions
\ No newline at end of file
+/functions
+/.netlify/functions*
```

Comment:
can we keep these paths under the `build` directory so that users don't have to modify their `.gitignore` from the default `.gitignore` provided with new projects?

Reply:
These are the directories created by the Netlify CLI whenever it is run, so we can't change where they are. The CLI adds a `.gitignore` line when it's run if there isn't one already. By default it will ignore the whole directory, but as the plugin is in there now in the demo site, I added a line to just ignore the generated functions directories. Users won't need to add this.

Reply:
Do we need both `/functions` and `/netlify/functions` in the `.gitignore` in that case?

Reply:
True. There's no need for `/functions` anymore.